<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FabReport</class>
 <widget class="QWidget" name="FabReport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1176</width>
    <height>787</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="4">
    <widget class="QTableWidget" name="twMassReport"/>
   </item>
   <item row="2" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>1067</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="bnSave">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/mnemo/Save-icon.png</normaloff>:/mnemo/Save-icon.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="bnPrint">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/mnemo/printer.png</normaloff>:/mnemo/printer.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QTableWidget" name="twReport"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="mnemo/mnemo.qrc"/>
 </resources>
 <connections/>
</ui>
